app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,27 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
+  .govuk-breadcrumbs__list {
+    display: flex;
+  }
+
+  .govuk-breadcrumbs__list-item {
```

Comment:
You could consider `.govuk-breadcrumbs__list-item:not(:last-child):not(:first-child)` here.